src/Components/CreateImageWizardV2/utilities/requestMapper.tsx

```diff
@@ -8,6 +8,7 @@ import {
   CreateBlueprintRequest,
   Customizations,
   DistributionProfileItem,
+  Filesystem,
```

Comment:
```suggestion
  FileSystem,
```

Reply:
I agree we should use `FileSystem` camel case but this is an import from the `image-builder` API so I'll suggest we keep it and don't rename it.